dspace/config/registries/openaire-cerif-types.xml

```diff
@@ -0,0 +1,232 @@
+        <element>oamandate</element>
+        <qualifier>url</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>project</element>
+        <qualifier>startDate</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>project</element>
+        <qualifier>endDate</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>project</element>
+        <qualifier>status</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>fundingProgram</element>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>fundingParent</element>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>amount</element>
+        <qualifier>currency</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>amount</element>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>funding</element>
+        <qualifier>identifier</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>funding</element>
+        <qualifier>startDate</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>citation</element>
+        <qualifier>number</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>funding</element>
+        <qualifier>endDate</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>event</element>
+        <qualifier>startDate</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>event</element>
+        <qualifier>endDate</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>event</element>
+        <qualifier>place</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>event</element>
+        <qualifier>country</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+<!-- These below should be moved in the oairecerif schema -->
+    <dc-type>
+        <schema>dc</schema>
+        <element>relation</element>
+        <qualifier>project</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
```

Comment:
These fields seem like they are in the wrong file.  You are adding new dc.relation fields.

It's also unclear what these fields mean and how they should be used (they need scope notes to describe).  Is "dc.relation.project" supposed to be a UUID of a related Project?  Or maybe a String?  Or maybe a URI?  What's the purpose?   And how does this differ from the `relation.isProjectOfPerson` (and similar) fields that we already have for Entity relationships?